src/main/java/core/basesyntax/ShopServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+import java.util.List;
+
+public class ShopServiceImpl implements ShopService {
```

Comment:
According to the checklist, logic should not be kept in a single package. Ensure that this service class is in the correct package, such as `service.impl` to follow the project structure guidelines.